crates/sui-types/src/object.rs

```diff
@@ -466,7 +465,7 @@ impl Data {
 }
 
 #[derive(
-    Eq, PartialEq, Debug, Clone, Copy, Deserialize, Serialize, Hash, JsonSchema, Ord, PartialOrd,
+    Eq, PartialEq, Debug, Clone, Deserialize, Serialize, Hash, JsonSchema, Ord, PartialOrd,
```

Comment:
I think its sort of sad to see the loss of Copy here, meaning this type got more complex. I wonder if it would be possible to tweak the format here to preserve that?

Reply:
Do you have any recommendations on how to do that without making the type bigger?

Reply:
we could just get rid of the Box<> now, right? It's not necessary to fix the tests.  So the question is whether it's worth paying 8 bytes on every single Object allocation in order to avoid adding one 32 byte allocation to every ConsensusV2 object?

I doubt either way will make any measureable difference right now, so removing the Box<> might be nice just for ergonomic reasons?

Reply:
I think there is a good chance we will want to `Box` future authenticators that may be larger than the existing single variant, so since I already did the work to remove `Copy` maybe we can keep it unless there's a strong reason not to?

for that matter maybe we'd want to add Box to existing variants? (not that I'll be doing it in this PR)